web/themes/custom/votegov/templates/node/node--state-territory.html.twig

```diff
@@ -118,21 +111,22 @@
       {{ registration_intro.text }}
     {% endif %}
 
-    {% if has_online and (online_registration is not empty) %}
-      {{ online_registration }}
-      {{ online_deadline }}
-      {{ registration_link }}
-      {% set body = online_registration.text['#markup'] | t({'@state_online_deadline': online_deadline | render, '@state_name': title_english}) %}
-      {% set link_title = online_registration.link_text['#markup'] | t({'@state_name': title_english }) %}
-      {% include '@votegov/component/info-card.html.twig' with {
-        'heading': online_registration.heading,
-        'body': body,
-          'link': {
-            'url': registration_link,
-            'title': link_title,
-          }
-      } %}
-    {% endif %}
+{# setting online registration variables #}
```

Comment:
i set this up to use variables since both options are working the same way just outputting different content